scenarioo-client/app/diffViewer/diffInfo.service.ts

```diff
@@ -1,70 +1,57 @@
-/* scenarioo-client
- * Copyright (C) 2014, scenarioo.org Development Team
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
```

Comment:
At the moment it looks like this header should be present in all files. There is an open issue, which we want to address for release 6.0, which will add this header to all files that do not have them yet. You don't have to add it here at the moment. But if you encounter other files, it would be easier to just leave it in place. See issue #431 for more details.